.github/workflows/cve-scanning.yml

```diff
@@ -0,0 +1,50 @@
+name: Node.js CVE Scanning
+
+on:
+  pull_request:
+    paths:
+      - 'package.json'
+      - 'toolbox/fdc3-workbench/package.json'
+      - '.github/workflows/cve-scanning.yml'
+      - 'website/package.json'
+  push:
+    paths:
+      - 'package.json'
+      - 'toolbox/fdc3-workbench/package.json'
+      - '.github/workflows/cve-scanning.yml'
+      - 'website/package.json'
+    schedule:
+      # Run every day at 5am and 5pm
+      - cron: '0 5,17 * * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x]
```

Comment:
Ideally, change this to whatever is necessary to ensure >= 16.17.0 as that will ensure we get npm 8.15.0 rather than 8.11.0. The audit for workbench will come back clean for npm >= 8.15.0.

We'll probably get >=16.17.0 anyway as its the latest LTS, but might be best to ensure it.